src/olmo_core/data/mixes/__init__.py

```diff
@@ -7,10 +7,19 @@
 
 from ..tokenizer import TokenizerName
 
-__all__ = ["DataMix"]
+__all__ = ["BaseDataMix", "DataMix"]
```

Comment:
Can you also export `BaseDataMix` in `src/olmo_core/data/__init__.py`?

Reply:
Done